airgun/entities/ansible_role.py

```diff
@@ -37,12 +37,7 @@ def delete(self, entity_name):
     def imported_roles_count(self):
         """Return the number of Ansible roles currently imported into Satellite"""
         view = self.navigate_to(self, 'All')
-        # Before any roles have been imported, no table or pagination widget are
-        # present on the page
-        if not view.pagination.is_displayed:
```

Comment:
why are you removing this check?

Reply:
At least when I've been debugging the test, It's not properly respecting the actual pagination, and the test runs find without it.

Reply:
I think the comment here provides valuable context, and I think that this property should return `0` if no roles have been imported. Without this check, the property returns an error if no roles have been imported. Consequently, I would request that you not remove these lines.

However, you're right that the check is not respecting the actual pagination. `view.pagination.is_displayed` returns false even when the pagination widget is present on the actual page. It looks to me like the pagination implementation has changed since I wrote this entity originally. I think the move here should be to address that discrepancy rather than remove this check. 

Looking at the `Pagination` widget [1], it's using `.//span[contains(@class, 'pagination-pf-items-total')]` as the locator for `total_items` (the same locator that you've changed to `//span[@class='pf-c-options-menu__toggle-text']/b[2]` in this PR) and `.//form[contains(@class, 'content-view-pf-pagination')]` as the `ROOT` locator for the widget. Looking at the pagination section in the HTML on the Ansible Roles page after importing roles on Satellite 6.12, I don't see either of those classes in use. The pagination widget is now in a `<foreman-react-component>` element instead of a `<form>` element. Looking at Satellite 6.11, the pagination widget HTML matches the locators currently defined in the Airgun `Pagination` widget.

So we need to refactor the `Pagination` widget for Satellite 6.12. Doing so may be outside the scope of this PR, so if you want to find an alternate way to check for the presence of the pagination widget as a temporary workaround, I think that would be acceptable as well.

[1] https://github.com/SatelliteQE/airgun/blob/007fe8bac68a980cd0ba5555ea535afda93b05df/airgun/widgets.py#L1586